src/exampleTypes/ResultTable.js

```diff
@@ -110,143 +113,171 @@ let HighlightedColumn = observer(
     )
   }
 )
-HighlightedColumn.displayName = 'HighlightedColumn'
 
-let HeaderCellDefault = observer(({ activeFilter, style, children }) => (
+let HeaderCellDefault = _.flow(
+  setDisplayName('HeaderCell'),
+  observer
+)(({ activeFilter, style, children }) => (
   <th style={{ ...(activeFilter ? { fontWeight: 900 } : {}), ...style }}>
     {children}
   </th>
 ))
-HeaderCellDefault.displayName = 'HeaderCellDefault'
 
-let Header = observer(({ // Local State
-  Modal, FieldPicker, ListGroupItem: Item, HeaderCell = HeaderCellDefault, field: fieldSchema, includes, addOptions, addFilter, tree, node, mutate, criteria, mapNodeToProps, fields, visibleFields, Icon }) => {
-  // Components (providerable?) // Contextual
-  let popover = useLens(false)
-  let adding = useLens(false)
-  let filtering = useLens(false)
-  let {
-    disableFilter,
-    disableSort,
-    field,
-    label,
-    hideMenu,
-    typeDefault,
-  } = fieldSchema
-  HeaderCell = fieldSchema.HeaderCell || HeaderCell
-  let filterNode =
-    criteria &&
-    _.find({ field }, _.getOr([], 'children', tree.getNode(criteria)))
-  let filter = () => {
-    if (!filterNode) addFilter(field)
-    filterNode =
+let Header = _.flow(
+  setDisplayName('Header'),
+  observer,
+  withTheme
+)(
+  ({
+    HeaderCell = HeaderCellDefault,
```

Comment:
If we can confirm nothing uses this, let's just kill it. We can always add it back in 2.0.1 if we need to